source/enumerate-databases.rst

```diff
@@ -190,7 +203,7 @@ MongoDatabase through MongoClient (e.g. read preference, write concern).
 Drivers SHOULD specify the ``nameOnly`` option when executing the
 ``listDatabases`` command for this method.
 
-Drivers SHOULD support the ``filter`` and ``authorizedDatabases`` options when implementing this method. 
+Drivers SHOULD support the ``filter`` and ``authorizedDatabases`` options when implementing this method.
```

Comment:
Should the optional `listDatabaseNames` and `listMongoDatabases` methods not also support `comment`?

Reply:
I think they should. Added `comment` option to corresponding sections.